File: backend/PolyPokerServer/src/main/java/com/beathuntercode/polypokerserver/websocket/messages/BetMessageContent.java
package com.beathuntercode.polypokerserver.websocket.messages;

public class BetMessageContent extends MessageContent {

    private int betValue;

    public BetMessageContent(int betValue) {
        this.betValue = betValue;
    }

    public int getBetValue() {
        return betValue;
    }

    public void setBetValue(int betValue) {
        this.betValue = betValue;
    }
}

File: android/PolyPoker/app/src/main/java/com/example/polypoker/websocket/stomp/messages/BetMessageContent.kt
package com.example.polypoker.websocket.stomp.messages

import com.example.polypoker.model.CardNumber
import com.example.polypoker.model.CardSuit

class BetMessageContent (
    roomCode: Int,
    private var betValue: Int
) : MessageContent(roomCode) {

}